lua/octo/init.lua

```diff
@@ -444,9 +471,13 @@ function M.create_issue_buffer(issue, repo, create_buffer)
   api.nvim_buf_set_var(bufnr, "labels", issue.labels)
   api.nvim_buf_set_var(bufnr, "assignees", issue.assignees)
   api.nvim_buf_set_var(bufnr, "milestone", issue.milestone)
+  api.nvim_buf_set_var(bufnr, "taggable_users", {issue.user.login})
 
   -- local mappings
-  local mapping_opts = {script = true, silent = true}
+  local mapping_opts = {script = true, silent = true, noremap = true}
+
+  api.nvim_buf_set_keymap(bufnr, "i", "@", "@<C-x><C-o>", mapping_opts)
+  api.nvim_buf_set_keymap(bufnr, "i", "#", "#<C-x><C-o>", mapping_opts)
```

Comment:
that is a great idea, thanks!

Reply:
`#` being a common character in markdown is a good point. I think there's a couple options:
  - Refactor fetching issues to happen in the background and attach to the buffer like users
  - Keep the fetching as-is but also add an `nvim_buf_(get/set)_var` in the completion function that only fetches if the value is nil
  - Remove these mappings so the user has to purposely try to fetch issues/users
  
I'm probably more of a fan of fetching the issues like the users and attaching them to the buffer, but I'll defer to you.

Reply:
That looks good to me and solve the cache problem, so Im ok with it

Reply:
I'll get started on that now :+1:

Reply:
Maybe use a global dict instad of a buffer var so it can be reused by other issue buffers from the same repo

Reply:
I think that's a possibility but I wonder if that's too wide. The issues that you can auto-complete will be defined by the time you open the first issue buffer. So if you want to complete issues that are created later you would need to restart nvim/manually clear that global. Unless we also introduce some cache invalidation?

It may also be worth mentioning that I think a user should be able to make 5000 requests per hour with an api token if you're worried about that.
https://docs.github.com/en/free-pro-team@latest/developers/apps/rate-limits-for-github-apps#normal-user-to-server-rate-limits

Reply:
Ok,that makes sense, lets make it a buf var then